zerver/tests/test_messages.py

```diff
@@ -398,7 +403,7 @@ def test_non_ascii_personal(self):
         Sending a PM containing non-ASCII characters succeeds.
         """
         self.login("hamlet@zulip.com")
-        self.assert_personal("hamlet@zulip.com", "othello@zulip.com", u"hümbüǵ")
+        self.assert_personal("hamlet@zulip.com", "zulip", "othello@zulip.com", "zulip", u"hümbüǵ")
```

Comment:
I think a cleaner solution to this might be to use hardcode "zulip" in `assert_personal`.